framework/bigcommerce/common/get-site-info.ts

```diff
@@ -94,11 +96,20 @@ async function getSiteInfo({
     query,
     { variables }
   )
-  const categories = data.site?.categoryTree
+
+  let categories = data!.site!.categoryTree?.map(
+    ({ entityId, name, path }: any) => ({
+      id: `${entityId}`,
+      name,
+      slug: getSlug(path),
+      path,
+    })
+  )
```

Comment:
Let's make this a normalize function, like the one we have for cart data, and add a comment with a TODO on top of the `any` type, we have BC types that should be able to match here.